### PR TITLE
Big CSters Blogging Workshop

### PR DESCRIPTION
**Pull request by julianguyen**

### CODE REVIEW

**Comment by ehashman** [comment]
Is this in reference to https://git.uwaterloo.ca/wics/documentation/issues/55 and does this event put a hard deadline on that issue?

File: content/W2016/W16-event-blogging.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Title: Big CSters Blogging Workshop
+Date: 2016-02-23 19:00
+Category: Events
+Tags: social, big csters, blogging
+Slug: blogging-workshop
+Author: Julia Nguyen
+Summary: Join us for an interactive workshop on blogging!
+
+Interested in technical writing, blogging, or crafting your online identity? Join us for an interactive workshop on blogging! This workshop will help you hone in on your writing - from generating ideas to finding the right publishing platform! No experience is required. Like every Big CSters event, dinner will be served!
```
Comment (ehashman) [inline content/W2016/W16-event-blogging.md]: Hone in on -> hone?

**Comment by julianguyen** [comment]
@ehashman This is my first time seeing that issue! Looks promising. I was talking about discussing whether to self-publish (on personal website, Medium, WordPress, Blogger etc.) or publish for a publication (school newspaper, websites that feature bloggers, etc.). Updated the post!

File: content/W2016/W16-event-blogging.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Date: 2016-02-23 19:00
+Category: Events
+Tags: social, big csters, blogging
+Slug: blogging-workshop
+Author: Julia Nguyen
+Summary: Join us for an interactive workshop on blogging!
+
+Interested in technical writing, blogging, or crafting your online identity? 
+Join us for an interactive workshop on blogging! This workshop will help you hone your writing 
+- from generating ideas to finding the right publishing platform!
```
Comment (ehashman) [inline content/W2016/W16-event-blogging.md]: Now that this line has been split, starting it with a `-` renders this as a bullet point (see https://github.com/julianguyen/wics-website/blob/master/content/W2016/W16-event-blogging.md)

**Comment by ehashman** [comment]
Re: the WiCS blogging platform, can you please provide input on that issue?

File: content/W2016/W16-event-blogging.md (hunk 2 above)
Comment (julianguyen) [inline content/W2016/W16-event-blogging.md]: oh shoot let me fix that

**Pull request merged**